server/server.go

```diff
@@ -455,13 +455,14 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	username := slashCommand.Username
+	msg := fmt.Sprintf("`@%s` triggered a plugin release process.\nTag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", username, tag)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
 		if !force {
-			WriteErrorResponse(w, NewError(fmt.Errorf("Tag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", tag).Error(), nil))
+			WriteErrorResponse(w, NewError(fmt.Errorf("Username: `@%s`\nTag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", username, tag).Error(), nil))
```

Comment:
I not sure if `Username:` is really needed. A "normal" @ mention is the idiomatic way.
```suggestion
			WriteErrorResponse(w, NewError(fmt.Errorf("`@%s` Tag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", username, tag).Error(), nil))
```